### PR TITLE
Attach KHR_texture_transform to other extensions

### PR DESCRIPTION
**Pull request by lilleyse**
Partially addresses https://github.com/CesiumGS/cesium-native/issues/801.

Updates the generator code to register `KHR_texture_transform` on `ExtensionExtFeatureMetadataFeatureIDTexture` and `PropertyTextureProperty` but does not update `FeatureIdTextureView` or `PropertyTexturePropertyView`.

### CODE REVIEW

File: tools/generate-classes/index.js [hunk 1]
```diff
@@ -77,6 +79,16 @@ for (const schemaPath of argv.schemas) {
...
 const schemaCache = new SchemaCache(schemaBasePaths, argv.extensions);
 
+// Add extensions to schema cache so that extensions can extend objects in other extensions
+for (const extension of config.extensions) {
+  const extensionSchema = schemaCache.loadExtension(
+    extension.schema,
+    extension.extensionName
+  );
+  const { _, schemaBasePath } = splitSchemaPath(extensionSchema.sourcePath);
+  schemaCache.schemaPaths.push(schemaBasePath);
```
Comment (lilleyse) [inline tools/generate-classes/index.js:89]: I don't really like how this is modifying `schemaPaths`, but this approach didn't require too many code changes.

**Comment by j9liu** [comment]
@lilleyseshould this extend `FeatureIdTexture` for `EXT_mesh_features` too?  I see that this extends the legacy `EXT_feature_metadata` texture but not the newer one

**Comment by lilleyse** [comment]
Oh yes it should. Thanks for catching that.

**Comment by lilleyse** [comment]
Hm, so the problem is that `featureIdTexture.schema.json` is in both extensions and it ends up picking whichever one was added to the schema base paths first...

At this point maybe we can just drop support for the older extensions?

**Comment by j9liu** [comment]
Hm that's probably fine! No one has complained about it not being supported in our clients (yet?)

**Comment by lilleyse** [comment]
I opened a PR to remove EXT_feature_metadata: https://github.com/CesiumGS/cesium-native/pull/812. Once that's merged I'll update this PR.

File: tools/generate-classes/index.js (hunk 1 above)
Comment (j9liu) [reply tools/generate-classes/index.js:89]: @kring Do you have any concerns with this change? If not, happy to merge this. Just wanted to double-check

Comment (kring) [reply tools/generate-classes/index.js:89]: No concerns, it seems reasonable enough to me.

**Comment by kring** [comment]
Merging this in order to move on to #811. Thanks @lilleyse! And thanks @j9liu for all the review work.

**Pull request merged**